services/transcribestreaming/src/it/java/software/amazon/awssdk/services/transcribestreaming/TestResponseHandlers.java

```diff
@@ -0,0 +1,81 @@
+package software.amazon.awssdk.services.transcribestreaming;
+
+import static software.amazon.awssdk.core.http.HttpResponseHandler.X_AMZN_REQUEST_ID_HEADER;
+
+import software.amazon.awssdk.core.async.SdkPublisher;
+import software.amazon.awssdk.services.transcribestreaming.model.StartStreamTranscriptionResponse;
+import software.amazon.awssdk.services.transcribestreaming.model.StartStreamTranscriptionResponseHandler;
+import software.amazon.awssdk.services.transcribestreaming.model.TranscriptEvent;
+import software.amazon.awssdk.services.transcribestreaming.model.TranscriptResultStream;
+
+public class TestResponseHandlers {
+
+    /**
+     * A simple consumer of events to subscribe
+     */
+    public static StartStreamTranscriptionResponseHandler responseHandlerBuilder_Consumer() {
+        return StartStreamTranscriptionResponseHandler.builder()
+                                                      .onResponse(r -> {
+                                                          String idFromHeader = r.sdkHttpResponse()
+                                                                                 .firstMatchingHeader(X_AMZN_REQUEST_ID_HEADER)
+                                                                                 .orElse(null);
+                                                          System.out.println("Received Initial response: " + idFromHeader);
```

Comment:
log instead of print?

Reply:
Fixed

Reply:
Accidentally updated the old commit, sorry about that. The only changes are replacing System.out with log statements